auditorium/views/main.js

```diff
@@ -160,32 +175,28 @@ function view (state, emit) {
 
   var manage = !isOperator && state.model.allowsCookies
     ? html`
-      <h4>${__('Manage your data')}</h4>
-      <div class="row">
-        <div class="six columns">
-          <button class="btn u-full-width" data-role="optout" onclick="${handleOptout}">
+      <div class="card col">
+        <h4>${__('Manage your data')}</h4>
+          <a class="btn primary" data-role="optout" onclick="${handleOptout}">
             ${state.model.hasOptedOut ? __('Opt in') : __('Opt out')}
-          </button>
-        </div>
-        <div class="six columns">
-          <button class="btn u-full-width" data-role="purge" onclick="${handlePurge}">
+          </a>
+          <a class="btn primary" data-role="purge" onclick="${handlePurge}">
             ${__('Delete my data')}
-          </button>
+          </a>
         </div>
-      </div>
     `
     : null
 
-  var withSeparators = [accountHeader, rangeSelector, usersAndSessions, chart, pages, referrers, manage]
-    .filter(function (el) {
-      return el
-    })
-    .map(function (el) {
-      return html`${el}<hr>`
-    })
+  var rowPagesReferrersManage = html`
+      <div class="row">
+        ${pages} ${referrers} ${manage}
+      </div>
+    `
+
+  var withoutSeparators = [accountHeader, rowRangeUsersSessions, chart, rowPagesReferrersManage]
```

Comment:
Now that the `hr` is not used anymore, using something "separators" in the variable is slightly confusing (it implies there is still something "with" around). Something like "orderedSections" might be clearer to the reader about what this is doing.

Reply:
(y)